contracts/smart-wallet/dynamic/DynamicAccountFactory.sol

```diff
@@ -27,7 +27,7 @@ contract DynamicAccountFactory is BaseAccountFactory, ContractMetadata, Permissi
     //////////////////////////////////////////////////////////////*/
 
     constructor(IEntryPoint _entrypoint, IExtension.Extension[] memory _defaultExtensions)
-        BaseAccountFactory(payable(address(new DynamicAccount(_entrypoint, _defaultExtensions))))
+        BaseAccountFactory(payable(address(new DynamicAccount(_entrypoint, _defaultExtensions))), address(_entrypoint))
```

Comment:
why do you need the entrypoint address separately? can you not get it from the passed in `IEntrypoint` ?

Reply:
Yep, `address(_entrypoint)` is indeed coming from the passed in `IEntryPoint _entrypoint`.